app/controllers/students_controller.rb

```diff
@@ -1,6 +1,7 @@
 # Controller for Students
 class StudentsController < ApplicationController
   def index
-    render json: {hello: "Hello"}
+    student_list = Student.all
+    render json: student_list, status: :ok
```

Comment:
🎱  Later I would look into following a JSON spec like: https://jsonapi.org/
This will ensure you're following best practices and have consistency with your API endpoints. You'll probably want to look into using a JSON serializer too. (I wouldn't let this block you though and it makes sense to pump out more features.

Reply:
I think I am going to avoid this rabbithole for now.  jsonapi looks a little too complex for my needs.  But I should be looking more closely at my schema and making *something* consistent.